.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
- [ ]  Im glad to see you added a `gitignore` with node_modules, however all files that aren't relevant to the project should be placed in the `.gitignore` file. Please go ahead and add the `test.md`, ` hintrc`, `stylelintrc.json` and `package.json`  to the .gitignore file.